control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -1182,6 +1182,10 @@ func (r *reconciler) reconcileAuthOIDC(ctx context.Context, hcp *hyperv1.HostedC
 		// Copy OIDCClient Secrets into openshift-config namespace
 		if len(hcp.Spec.Configuration.Authentication.OIDCProviders[0].OIDCClients) > 0 {
 			for _, oidcClient := range hcp.Spec.Configuration.Authentication.OIDCProviders[0].OIDCClients {
+				// If the secret name is empty, we assume the guest cluster admin is going to create this secret manually
+				if oidcClient.ClientSecret.Name == "" {
+					continue
```

Comment:
I would add something like:

```
log.Info("OIDC client secret name is empty, skipping copy. Expected to be created manually.", "component", oidcClient.ComponentName)
```
to warn the user about the potential misconfiguration

Reply:
Not against it, but this isn't visible to the user.  It would just be in the HCCO logs on the mgmt cluster.